include/net/socket.h

```diff
@@ -208,6 +208,14 @@ __syscall int z_zsock_getaddrinfo_internal(const char *host,
 					   const struct zsock_addrinfo *hints,
 					   struct zsock_addrinfo *res);
 
+#define AI_PASSIVE 1
```

Comment:
What about documenting the macros?

Reply:
> What about documenting the macros?

I've added a header comment explaining what these are for getaddrinfo() (I also try to define them close to function as an extra hint). Documenting individual bits of Sockets API is covered by https://github.com/zephyrproject-rtos/zephyr/issues/13397 . So far, it's unclear how to approach it (e.g., given that it's "well-known API", one way would be to link to a standard describing it).

Reply:
lets start adding documentation when code is being added please. The bug you mention will address the remaining items.

Reply:
How can we start adding documentation if we haven't decided how to add it for this stuff? Actually, if we push someone too hard, I'm sure they can bend and copy-paste docs from some copyrighted material, paving the road for a SCO case in some 10 years (if Zephyr will ever get off the ground).

Reply:
please stop making a big fuss about documenting simple macros. We are not trying to come up with a standard here.

> Actually, if we push someone too hard, I'm sure they can bend and copy-paste docs from some copyrighted material, paving the road for a SCO case in some 10 years 

Sigh.

> if Zephyr will ever get off the ground.

With this kind of attitude probably never.

Reply:
> With this kind of attitude probably never.

Exactly. Either we're focused on making all that stuff *work* in 1.14, or we stop and make a ceremony of documenting non-working things.